3rdParty/rocksdb/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ set(ARANGO_ROCKSDB_VERSION "v5.6.X")
 # jemalloc settings
 if (USE_JEMALLOC)
   if (WIN32)
+      set(WITH_MD_LIBRARY OFF) #overwrite option in subproject -- libraries should not touch this (/MD /MT) at all!
```

Comment:
This does help for existing builds.

Also the "option" in rocksdb makes it a cache variable so it should be set to CACHE (if I am not mistaken).

Also set it CACHE FORCE so that it is non overridable by the user.